packages/map/src/components/UsaMap/helpers/map.ts

```diff
@@ -0,0 +1,100 @@
+  return `https://www.cdc.gov/TemplatePackage/contrib/data/county-topography/cb_${year}_us_county_20m.json`
+}
+
+export const getTopoData = year => {
+  return new Promise((resolve, reject) => {
+    const resolveWithTopo = async response => {
+      if (response.status !== 200) {
+        response = await import('./../data/cb_2019_us_county_20m.json')
+      } else {
+        response = await response.json()
+      }
+      let topoData = {}
+
+      topoData.year = year || 'default'
+      topoData.fulljson = response
+      topoData.counties = feature(response, response.objects.counties).features
+      topoData.states = feature(response, response.objects.states).features
+
+      resolve(topoData)
+    }
+
+    const numericYear = parseInt(year)
+
+    if (isNaN(numericYear)) {
+      fetch(getCountyTopoURL(2019)).then(resolveWithTopo)
+    } else if (numericYear > 2022) {
+      fetch(getCountyTopoURL(2022)).then(resolveWithTopo)
+    } else if (numericYear < 2013) {
+      fetch(getCountyTopoURL(2013)).then(resolveWithTopo)
+    } else {
+      switch (numericYear) {
+        case 2022:
+          fetch(getCountyTopoURL(2022)).then(resolveWithTopo)
+          break
+        case 2021:
+          fetch(getCountyTopoURL(2021)).then(resolveWithTopo)
+          break
+        case 2020:
+          fetch(getCountyTopoURL(2020)).then(resolveWithTopo)
+          break
+        case 2018:
+        case 2017:
+        case 2016:
+        case 2015:
+          fetch(getCountyTopoURL(2015)).then(resolveWithTopo)
+          break
+        case 2014:
+          fetch(getCountyTopoURL(2014)).then(resolveWithTopo)
+          break
+        case 2013:
+          fetch(getCountyTopoURL(2013)).then(resolveWithTopo)
+          break
+        default:
+          fetch(getCountyTopoURL(2019)).then(resolveWithTopo)
+          break
+      }
+    }
+  })
+}
+
+export const getCurrentTopoYear = (state, runtimeFilters) => {
+  let currentYear = state.general.countyCensusYear
+
+  if (state.general.filterControlsCountyYear && runtimeFilters && runtimeFilters.length > 0) {
+    let yearFilter = runtimeFilters.filter(filter => filter.columnName === state.general.filterControlsCountyYear)
+    if (yearFilter.length > 0 && yearFilter[0].active) {
+      currentYear = yearFilter[0].active
+    }
+  }
+
+  return currentYear || 'default'
+}
+
+export const isTopoReady = (topoData, state, runtimeFilters) => {
+  let currentYear = getCurrentTopoYear(state, runtimeFilters)
+
+  return topoData.year && (!currentYear || currentYear === topoData.year)
+}
+
+export const getFilterControllingStatePicked = (state, dashboardConfig) => {
+  debugger
+  if (!dashboardConfig?.dashboard?.sharedFilters) {
+    if (state.general.filterControlsStatePicked === '' && !state.general.statePicked.stateName) {
+      return 'Alabama'
+    } else {
+      return state.general.statePicked.stateName
+    }
+  }
+  const filters = dashboardConfig?.dashboard?.sharedFilters
```

Comment:
dashboard filters should only filter data, not control visualizations in this way.